src/translations.qrc

```diff
@@ -3,5 +3,6 @@
         <file alias="en_US">lubuntu-update_en_US.qm</file>
         <file alias="es_ES">lubuntu-update_es_ES.qm</file>
         <file alias="zh_CN">lubuntu-update_zh_CN.qm</file>
+        <file alias="zh_CN">lubuntu-update_fr_FR.qm</file>
```

Comment:
Please align this, and/or add any appropriate entries.

Reply:
Fixed.